pilot/pkg/autoregistration/controller.go

```diff
@@ -101,7 +91,7 @@ type Controller struct {
 	// do not support writing. We only use it here for reading WorkloadEntry/WorkloadGroup.
 	store model.ConfigStoreController
 
-	// Note: unregister is to update the workload entry status: like setting `DisconnectedAtAnnotation`
+	// Note: unregister is to update the workload entry status: like setting `annotation.IoIstioDisconnectedAt.Name`
```

Comment:
nit: comment with actual annotation name is more readable

Reply:
Yes, but just align with others

Reply:
OK, will restore